.github/workflows/Tests.yaml

```diff
@@ -322,6 +322,7 @@ jobs:
         use_pch: [ON]
         use_xsimd: [ON]
         charm_name: ["Charm 7.0.0"]
+        charm_build: [/work/charm_7_0_0/multicore-linux-x86_64-gcc]
```

Comment:
Delete this line, and also `charm_name` above. I believe this entry in the matrix is only needed if you want to _add_ builds to the matrix, not to modify an existing one. Instead, just set `CHARM_ROOT` for one of the builds and fall back to the default, like `BUILD_PYTHON_BINDINGS` below.